src/analytics_manager.cpp

```diff
@@ -742,4 +775,22 @@ void counter_event_t::serialize_as_docs(std::string &docs) {
     if (!docs.empty()) {
         docs.pop_back();
     }
+}
+
+bool AnalyticsManager::write_to_db(const nlohmann::json& payload) {
+    if(analytics_store) {
+        for(auto event: payload) {
+            std::string key = event["user_id"].get<std::string>() + "_" + std::to_string(event["timestamp"].get<uint64_t>());
```

Comment:
Length of unix timestamps can increase over time. We should instead store them as uin64 bytes. Check `StringUtils::serialize_uint64_t`.